src/core/helpers/http-status.ts

```diff
@@ -1,3 +1,26 @@
+export const OK = 200;
+export const CREATED = 201;
+export const ACCEPTED = 202;
+export const NO_CONTENT = 204;
+export const MOVED_PERMANENETLY = 301;
+export const SEE_OTHER = 303;
+export const NOT_MODIFIED = 304;
+export const TEMPORARY_REDIRECT = 307;
+export const INVALID_JSON = 400;
```

Comment:
It's fine to have `INVALID_JSON` as one of the `400` responses (that would be specific and idiomatic to Sakura). It is not ok to have `BAD_REQUEST` of `401` (see below) - `401` is `UNAUTHORIZED` and would never mean `BAD_REQUEST`.

Reply:
Upon reflection, I think it's best we don't have `INVALID_JSON`... it requires too much interpretation as to what that means and if the JSON itself is invalid (which is really what this should mean), then that's handled in one place by the body parser.

Reply:
The 400 error code was in various spec files.  i took the most common interpretation and called it INVALID_JSON.  400 isn't in the list from the #19 issue.

Reply:
removed INVALID_JSON.  400 is BAD_REQUEST and 401 is UNAUTHORIZED